sdk/communication/AzureCommunicationChat/AzureCommunicationChat.xcodeproj/project.pbxproj

```diff
@@ -136,13 +136,13 @@
 		D1FDDBE82672B73700F9D0B0 /* RegistrarSettings.swift in Sources */ = {isa = PBXBuildFile; fileRef = D10D5581266AB3F6009504FA /* RegistrarSettings.swift */; };
 		D1FDDBE92672B73A00F9D0B0 /* RegistrarRequestBody.swift in Sources */ = {isa = PBXBuildFile; fileRef = D1A0C36B26703FE900D6780A /* RegistrarRequestBody.swift */; };
 		D1FDDBEA2672B73D00F9D0B0 /* RegistrarClientDescription.swift in Sources */ = {isa = PBXBuildFile; fileRef = D1A0C36726703D5200D6780A /* RegistrarClientDescription.swift */; };
+		E0130F6827DFDC16005DA385 /* test-settings.plist in Resources */ = {isa = PBXBuildFile; fileRef = E0130F6727DFDC16005DA385 /* test-settings.plist */; };
```

Comment:
You need to remove this if you are going to get CI to pass.

Reply:
removed